pkg/api/explorer.go

```diff
@@ -387,7 +404,7 @@ func getUnconfirmedTransactions(explorer modules.Explorer, tpool modules.Transac
 		for _, bsi := range txn.BlockStakeInputs {
 			bsi, _ := explorer.BlockStakeOutput(bsi.ParentID)
 			if bsi.Condition.UnlockHash() == addr {
-				related = true
+				// related = true // ineffectassign
```

Comment:
does this need to say?

Reply:
It can stay, but then ineffectassign is not happy. It is anyhow not useful, as we do not consume the value any further.

Reply:
I Left it commented so it is obvious that should this logic be extended this variable is needed.

Reply:
Actually this code it silly to begin with, it should just label the outer loop and continue its next cycle immediately.